code401challenges/src/main/java/code401challenges/stacksandqueues/PseudoQueue.java

```diff
@@ -0,0 +1,25 @@
+package code401challenges.stacksandqueues;
+
+import java.util.NoSuchElementException;
+
+public class PseudoQueue<T> {
+    Stack<T> stackOne;
+    Stack<T> stackTwo;
+
+    public PseudoQueue() {
+        this.stackOne = new Stack<>();
+        this.stackTwo = new Stack<>();
+    }
+
+    public void enqueue(T value) {
+        this.stackOne.push(value);
+    }
+
+    public T dequeue() {

```

Comment:
This method doesn't quite work in all scenarios. If you have a `pseudoqueue` and you `enqueue()` `a`, `b` and `c`:
```
     c
     b
     a
firstStack      secondStack
```
Then we call `dequeue()` and we should get back `a` since `a` was the first value put into the `pseudoqueue`. This code will move `c` into `secondStack` and then return `b`.
```
     
     b <--
     a                      c
firstStack      secondStack
```